### DISCUSSION TITLE
Issue with Parallel Gateway

### DISCUSSION THREAD

**Discussion by SundaramMaheshkumar** [bpmnServer/bpmn-server#83] [General]
Hi @ralphhanna,

_Using version 1.3.15_

Parallel Gateway doesn't seem to wait for all its incoming events before progressing. 

![image](https://user-images.githubusercontent.com/118889895/230318956-b16ad756-91c7-4254-a627-262f4b967c0c.png)

It seems that there was an [issue ](https://github.com/ralphhanna/bpmn-server/issues/71)already raised regarding this and the comments indicate that the issue is resolved. But I still face this issue.

Could you please provide some clarification?

Thanks

**Comment by SundaramMaheshkumar** [comment]
Sorry, I messed up the workflow when I designed one. No issue here

**Comment by bpmn-ts** [comment]
HI
Please either test on BPMN.OmniWorkflow.com or attach your process
definition.

Thanks
